### PR TITLE
Update beta version usage and some fixes

### PR DESCRIPTION
**Pull request by addb**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: AEPEdgeMedia.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -1380,7 +1380,7 @@
 					"@executable_path/Frameworks",
 					"@loader_path/Frameworks",
 				);
-				MARKETING_VERSION = 1.0;
+				MARKETING_VERSION = "";
```
Comment (emdobrin) [inline AEPEdgeMedia.xcodeproj/project.pbxproj:1383]: why not using the same version here 1.0.0-beta?

File: Documentation/migration-guide.md [hunk 2]
```diff
@@ -96,7 +96,7 @@ import AEPCore
 - import AEPMedia
 + import AEPEdge
 + import AEPEdgeIdentity
-+ import AEPEdgeMedia
++ pod 'AEPEdgeMedia', :git => 'https://github.com/adobe/aepsdk-edgemedia-ios.git', :tag => '1.0.0-beta'
```
Comment (emdobrin) [inline Documentation/migration-guide.md:99]: incorrect replacement? let's revert this

File: AEPEdgeMedia.xcodeproj/project.pbxproj (hunk 1 above)
Comment (emdobrin) [reply AEPEdgeMedia.xcodeproj/project.pbxproj:1383]: I noticed that for some of these test targets we use IPHONEOS_DEPLOYMENT_TARGET = 15.5 and TVOS_DEPLOYMENT_TARGET= 15.0. Can we set that to min supported by the sdk 10.0?

Comment (emdobrin) [reply AEPEdgeMedia.xcodeproj/project.pbxproj:1383]: And for the `MARKETING_VERSION` can we remove that field altogether? The other projects have it only for the lib target, and we may use automation to update these fields which may incorrectly update these

Comment (addb) [reply AEPEdgeMedia.xcodeproj/project.pbxproj:1383]: I did not find ui option. I can just remove from the project file.

**Review by emdobrin — APPROVED** [review]

**Review by kevinlind — APPROVED** [review]

**Pull request merged**